src/unlinkcommand.js

```diff
@@ -14,6 +14,27 @@ import findLinkRange from './findlinkrange.js';
  */
 export default class UnlinkCommand extends Command {
 	/**
+	 * @see core.command.Command
+	 * @param {core.editor.Editor} editor
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		/**
+		 * Flag indicating whether command is active. For collapsed selection it means that typed characters will have
+		 * the command's attribute set. For range selection it means that all nodes inside have the attribute applied.
+		 *
+		 * @observable
+		 * @member {Boolean} core.command.ToggleAttributeCommand#value
+		 */
+		this.set( 'hasValue', undefined );
```

Comment:
This is something that we discussed a very long time ago, when defining first commands. We wanted to have `isEnabled` and `isOn`. But then we realised that you have features like heading or font color which state is more complex. So we have `isEnabled` and `value` which can have different type of values, depending on a command.

Now, for me we can proceed this way OR make `isOn` and `value(s)` separate. I remember that we've had a discussion how image extensions (cause an image caption, alignment, etc are separate features) could add their values to the base image feature and one of the ideas were to simply use a more reach value field. Buut TBH, I don't know what will work for that case.

I'd proceed now with what we have in other places and perhaps discuss this somewhere else one day.

HOWEVER, why is this property called `hasValue` and not simply `value`? Does it represent something else?